params/params.go

```diff
@@ -262,6 +272,20 @@ func (p *Params) HasTax() bool {
 	return false
 }
 
+func (p *Params) IsValidTxType(tt types.TxType) bool {
```

Comment:
the judgement for validTxType should not couple with `params`. 
the possible flow :
 1. initialized network params config: `AccpectNonStd` is `false` . ignore the token-db etc when node startup.
     - if `TxType` is nostd  refuse the tx. 
     - if `TyType` is std tx, handle std tx.
 2. when network params config changed to : `AccpectNonStd` is `true`. setup toekn-db if not create before.
    -  if 'TxType' is nostd, check `ConsensusDeployment`, (ex : block order at 10000), if ok, handle nostd tx. else refuse tx. 
    - if 'TxType' is std tx, handle std tx.
 3.  the config `AccpectNonStd` from `true` changed to `false`.  the token-db existed in this case. node stop with error and ask user to do `--cleaup `, this case return to case 1.